docs/doc/50-manage/00-metasrv/30-metasrv-backup-restore.md

```diff
@@ -27,13 +27,13 @@ E.g., every line in the output file is a JSON of an exported key-value record.
 # ...
 ```
 
-##  Restore a databend-meta
+## Restore a databend-meta
 
 The following command rebuild a meta service db in `<your_meta_dir>` from
 exported metadata:
 
 ```sh
-cat "<output_fn>" | ./target/debug/databend-metactl --import --raft-dir "<your_meta_dir>"
+./target/debug/databend-metactl --import --raft-dir "<your_meta_dir>" --db <output_fn>
```

Comment:
could you elaborate on how could we restore a three node new cluster from one snapshot, ie
```bash
./target/debug/databend-metactl --import --raft-dir "<your_meta_dir1>" --db <output_fn>  --initial-cluster 1=localhost:28003
./target/debug/databend-metactl --import --raft-dir "<your_meta_dir2>" --db <output_fn>  --initial-cluster 2=localhost:28004
./target/debug/databend-metactl --import --raft-dir "<your_meta_dir3>" --db <output_fn>  --initial-cluster 3=localhost:28005

// start cluster
./target/debug/databend-meta --raft-dir "your raft dir1" --rpc-advertise-host localhost:28003 --id=1
./target/debug/databend-meta --raft-dir "your raft dir2" --rpc-advertise-host localhost:28004 --id=2
./target/debug/databend-meta --raft-dir "your raft dir3" --rpc-advertise-host localhost:28005 --id=3

```